.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Release a new version
+about: Start the process for a new release
+---
+
+# Release process
+
+This project uses [cargo-release][cargo-release] in order to prepare new releases, tag and sign relevant git commit, and publish the resulting artifacts to [crates.io][crates-io].
+The release process follows the usual PR-and-review flow, allowing an external reviewer to have a final check before publishing.
+
+## Requirements
+
+This guide requires:
+
+ * a web browser (and network connectivity)
+ * `git`
+ * GPG setup and personal key for signing
+ * `cargo` (suggested: latest stable toolchain from [rustup][rustup])
+ * `cargo-release` (suggested: `cargo install -f cargo-release`)
+ * A verified account on crates.io
+ * Write access to this GitHub project
+ * Upload access to this GitHub project
```

Comment:
Hmm, we don't upload binaries to GitHub for this project, right?

Reply:
I believe you're correct. Right now there is source tarball/zip files only.

Reply:
Good catch. Leftover from the `afterburn` one.